recipe/bowling_design_recipe.md

```diff
@@ -0,0 +1,318 @@
+# Bowling Class Design Recipe
+
+## 1. Describe the Problem
+
+_Put or write the user story here. Add any clarifying notes you might have._
+
+As a bowler
+So that I can find out if I won the game
+I want to see my total score at the end of game
+
+As a bowler
+So that I can see how many point I scored in a frame
+I want to see my total score after each frame
+
+As a bowler
+So that I can see how well I played
+I want to see how many strike and spares I scored at the end of a game
```

Comment:
This sounds like you have taken it further than the initial specification, which is cool!